shared/Cargo.toml

```diff
@@ -27,6 +27,8 @@ tokio-rustls = { version = "0.23.4", features = ["dangerous_configuration"] }
 sys-info = "0.9.1"
 cadence = "0.29.0"
 httparse = "1.8.0"
+aws-sdk-s3 = "0.28.0"
```

Comment:
Not for this line but slightly similar - we could pull the s3 sdk crate out of the control plane's `cargo.toml`